### ISSUE TITLE
一些小问题

### ISSUE THREAD

**Issue by realman520** [Jian-Lang/REDEEM#4]
你好！你的工作和开放的代码都非常棒！ 
但还存在一些小问题 或许可以修改一下
1.https://github.com/Jian-Lang/REDEEM/blob/master/src/model/modules.py 
  下
 def forward(self, rem_fea, ret_fea, **kwargs):
        routing_score = self.router(rem_fea, ret_fea)  # shape: [B, K]
        expert_outs = []
        for k in range(self.num_expert):
            expert_outs.append(self.experts[k](ret_fea[:, k]))  # [B, S, hs]
        expert_outs = torch.stack(expert_outs, dim=1)  # [B, K, S, hs]
        gen_fea = torch.einsum('bksd,bk->bsd', expert_outs, routing_score)
        print(gen_fea.size()) 这里会一直打印size  导致日志非常冗余 或许可以删去 
        return gen_fea

2.src/dataloader/__init__.py 这里或许要改成redeem 否则会报错 
from .ragpt_food101_dataset import Food101Dataset
from .ragpt_hatememes_dataset import HatememesDataset
from .ragpt_mmimdb_dataset import MMIMDbDataset

**Comment by Jian-Lang** [comment]
Thanks a lot for your attention to our work and suggestions. We will take time to carefully refine the source code.
